src/liquid/components/ld-notification/ld-notification.css

```diff
@@ -0,0 +1,159 @@
+.ld-notification {
+  position: fixed;
+  z-index: 2147483647; /* Highest possible */
+  display: flex;
+  justify-content: center;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  min-width: 20rem;
+  font: var(--ld-typo-body-m);
+  pointer-events: none;
+}
+
+@keyframes ld-notification-fade-in {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+@keyframes ld-notification-fade-out {
+  from {
+    transform: none;
+    opacity: 1;
+  }
+  to {
+    transform: none;
+    opacity: 0;
+  }
+}
+
+.ld-notification__item {
+  pointer-events: auto;
+  position: absolute;
+  transform-origin: center;
+  box-shadow: var(--ld-shadow-sticky);
+  width: fit-content;
+  min-width: 19rem;
+  max-width: calc(90% - 1rem);
+  max-height: 80vh;
+  min-height: 2.5rem;
+  overflow-y: auto;
+  border-radius: var(--ld-br-m);
+  display: flex;
+  justify-content: space-between;
+  transition: transform 0.2s ease, opacity 0.2s linear;
+
+  &:where(:not(.ld-notification__item--dismissed)) {
+    animation: ld-notification-fade-in 0.2s linear;
```

Comment:
This animation is re-run every time a new item is added to the stack - even if the new item is not shown on top. We probably should find a way to not re-render the items in the stack that actually don't change, because this also makes the animation of adding a new item below the first item feel more natural.

Reply:
I'm afraid this is not yet fixed... If you want to, I can show you what I mean in a call. :)

Reply:
Setting correct keys seems to resolve the issue. Can you check again on your side?

Reply:
Works like a charm!